platform/ios/INSTALL.md

```diff
@@ -95,7 +95,18 @@ If your application targets iOS 7.x, you’ll need to install the static framewo
 
 1. Build from source manually per above.
 
-1. Open the project editor and select your application target. Drag `build/ios/pkg/static/Mapbox.framework` into the “Embedded Binaries” section of the General tab. (Don’t drag it into the “Linked Frameworks and Libraries” section; Xcode will add it there automatically.) In the sheet that appears, make sure “Copy items if needed” is checked, then click Finish.
+1. Drag the Mapbox.bundle and Mapbox.framework files in `build/ios/pkg/static` into the Project navigator, checking "Copy items if needed". It should happen automatically, but ensure that:
+   
+   - `Mapbox.framework` is listed in your `Link Binary With Libraries` build phase.
+   - The path to the project's local copy of `Mapbox.framework` is in your *Framework Search Paths* (`FRAMEWORK_SEARCH_PATHS`) build setting.   
+   - `Mapbox.bundle` is in your target's *Copy Bundle Resources* build phase.
+
+<img width="322" alt="screen shot 2016-03-24 at 6 42 14 pm" src="https://cloud.githubusercontent.com/assets/63186/14035855/4744d476-f1f0-11e5-9392-6b19a5658835.png">
```

Comment:
The alt text should be more descriptive.

As for the image itself, I think it’d be better to link to the image on raw.githubusercontent.com, as we do on pod-README.md, rather than cloud.githubusercontent.com. The former breaks when the file is moved within the repository (but you can address that by pinning to a revision), while the latter breaks when GitHub garbage-collects images that are no longer needed for issues and comments.